rest-server/src/templates/yarnContainerScript.mustache

```diff
@@ -87,6 +87,7 @@ docker run --name $docker_name \
   --volume /tmp/pai-root/alive/$APP_ID:/alive \
   --label PAI_HOSTNAME="$(hostname)" \
   --label PAI_JOB_NAME={{{ jobData.jobName }}} \
+  --label PAI_JOB_QUEUE_NAME={{{ jobData.queue }}} \
```

Comment:
PAI_JOB_VC_NAME

Reply:
By the way, we need to have the following layers of concepts in mind:
- Layer 1: Virtual cluster;
- Layer 2: Job;
- Layer 3: Task role;
- Layer 4: Task.
So please organize your code lines in the order of above layers.